third_party/istio-1.3-latest

```diff
@@ -1 +1 @@
-istio-1.3.3
\ No newline at end of file
```

Comment:
<!--whitespace-bot-->
Add trailing newline:
```suggestion
istio-1.3.5

```

Reply:
As this is a symlink, it seems that we cannot add trailing newline(?).